packages/salesforcedx-utils/test/jest/requestService/__snapshots__/requestService.test.ts.snap

```diff
@@ -0,0 +1,86 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`RequestService unit tests. execute() Should include proxy authorization info if defined. 1`] = `
+[MockFunction sendRequestMock] {
+  "calls": Array [
+    Array [
+      Object {
+        "data": "this-is-a-test-request-body",
+        "headers": Object {
+          "Accept": "application/json",
+          "Authorization": "OAuth totallyfake-access-token-not-real-1234",
+          "Content-Length": 27,
+          "Content-Type": "application/json;charset=utf-8",
+          "Proxy-Authorization": "this-is-fake-proxy-auth",
+          "Sforce-Call-Options": "client=sfdx-vscode",
+        },
+        "timeout": 20000,
+        "type": "POST",
+        "url": "https://where.is/the/instance/https://this.is.a.test",
+      },
+    ],
+  ],
+  "results": Array [
+    Object {
+      "type": "return",
+      "value": Promise {},
+    },
+  ],
+}
+`;
```

Comment:
much more interesting snapshots here.  Note here it is useful b/c I don't have to check all the properties individually or in a constructed object in the call like headers, data, etc.